keycloak/keycloak-admin-ui
# Users(groups): list top level groups, implement direct membership check

## Motivation

#466 

## Brief Description


## Verification Steps


1. Go to the old admin console and create a group and another group with subgroups.
2. In the old admin console, go to the "Users" section, click on the "Groups" tab and add those groups to an existing user.
3. In the new admin console, go to the "Users" section and find the existing user.
4. Click on the User and go to the "Groups" tab.
5. Verify the joined groups show up in the table, and that the top-level groups are also visible.
6. Click on the "Direct membership" check and verify that the top-level groups disappear from the user view.

## Checklist:


## Additional Notes
